libsolidity/ast/AST.h

```diff
@@ -515,7 +515,7 @@ class ContractDefinition: public Declaration, public StructurallyDocumented, pub
 	std::map<util::FixedHash<4>, FunctionTypePointer> interfaceFunctions(bool _includeInheritedFunctions = true) const;
 	std::vector<std::pair<util::FixedHash<4>, FunctionTypePointer>> const& interfaceFunctionList(bool _includeInheritedFunctions = true) const;
 	/// @returns the EIP-165 compatible interface identifier. This will exclude inherited functions.
-	uint64_t interfaceId() const;
+	uint32_t interfaceId() const;
```

Comment:
No idea why this was uint64_t in the first place.

Reply:
If there is no reason for `uint64_t` looks good to merge